contents/spotlight/startup-puzzle.md

```diff
@@ -0,0 +1,79 @@
+date: 2024-02-01
+author: ["joe-martin"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/startup_puzzle.jpg
+featuredImageType: full
+category: Startups
+---
+
+> Each month, we speak to a team enrolled in PostHog for Startups about the way they work. This time, we spoke to [Puzzle](https://puzzle.io/), an accountancy platform for early-stage startups, about how they define and use their buyer persona. 
+
+Building a persona for your ideal or target customers is one of the most challenging but essential steps for growing your product. Not only will it help you make product and pricing decisions, it will also clarify your value proposition and dictate how you judge successes and failures.
+
+We’ve previously documented [how we defined our personas](/founders/creating-ideal-customer-profile), but not all companies approach the task in the same way. So, we spoke to the team at [Puzzle](https://puzzle.io/) about how they defined their persona, what impact the definition had, and what recommendations they had for others...
+
+## 1. Collect both qualitative and quantitative data
+
+A persona is, fundamentally, a snapshot of your ideal customer based on a broad understanding of many customers. In order to define your persona you therefore need to start by speaking to your existing users directly and building an intuitive understanding of their needs.
+
+Once you’ve done that, says Scott, you need to pair that intuitive understanding with quantitative data, so you can understand if these behaviors exist in isolation or at scale. 
+
+“We’re constantly standing next to founders who are just starting out and I do a healthy amount of interviews,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your ICP.”
+
+> **Putting it into practice:** It can help to [define a small number of reference customers](/founders/creating-ideal-customer-profile#5-reference-customers). Spend time with them to gather subjective data, while using a tool such as [PostHog](/startups) to collect hard facts.
+
+## 2. Understand how your users _feel_
+
+Armed with this qualitative and quantitative data, you can start to establish your value proposition.
+
+For Puzzle, this was critical for differentiating them from other products in the market. Every company needs an accounting platform, but not all founders feel the same about this need — and by focusing on users’ unaddressed needs Puzzle can identify a strong, unique position for the product.
+
+“Founders don’t sign up to Puzzle because they just can’t wait to do their accounting,” says Scott. “They sign up because they know they need to be tax compliant and to be ready for investor reporting... We learned that they care about efficiency, accuracy and, as they generate more revenue, delaying hires in the finance department.”
+
```

Comment:
```suggestion
“More often than not, founders aren’t signing up to Puzzle because they’re excited to do their accounting,” says Scott. “They sign up because they know they need to be tax compliant, gain a deeper understanding of their company’s finances, and be ready for things like investor reporting. While the needs of founders often change or mature at each stage of their journey, what’s common is that they care about compliance, efficiency, and accuracy.”

```